### PR TITLE
first anaconda build

### PR DESCRIPTION
**Pull request by mcg1969**
- commented out noarch
- added abs.yaml for testing
- simplified host dependencies down to pip/setuptools/wheel
- added description, doc_source_url
- removed python bound

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,31 +1,28 @@
...
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 907601c99f7510e16d27d7cb62bfa145216d166a2b5c9c50cfe2b65b032ebd2e
+  sha256: {{ sha256 }}
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  noarch: python
+  # noarch: python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:16]: Could you add skip `py<37`? https://github.com/visgl/deck.gl/blob/6708cd139680863d17335600066257a2d16c2c68/bindings/pydeck/pyproject.toml#L4

File: recipe/meta.yaml [hunk 2]
```diff
@@ -48,7 +45,11 @@ about:
   license_family: MIT
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: The license is Apache-2.0, see https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/LICENSE.txt
And `license_family: Apache`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,31 +1,28 @@
...
-    - ipython
-    - jinja2
+    - python 
     - pip
-    - python
-    - traitlets
-    - numpy >=1.16.4
+    - setuptools <61
+    - wheel
   run:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:26]: We need to update run dependencies, see https://github.com/visgl/deck.gl/blob/6708cd139680863d17335600066257a2d16c2c68/bindings/pydeck/setup.py#L213-L219

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,31 +1,28 @@
...
+  sha256: {{ sha256 }}
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  noarch: python
+  # noarch: python
 
 requirements:
   host:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:20]: Could you please add `jinja2` in `host`? See https://github.com/visgl/deck.gl/blob/6708cd139680863d17335600066257a2d16c2c68/bindings/pydeck/setup.py#L221

Comment (mcg1969) [reply recipe/meta.yaml:20]: I'm doing this now. Are you OK with me omitting the other setup dependencies, since we are using `pip --no-deps`?

**Comment by mcg1969** [comment]
Thank you @skupr-anaconda for the super-thorough review as always! Working on it

File: recipe/meta.yaml (hunk 1 above)
Comment (mcg1969) [reply recipe/meta.yaml:16]: done

File: recipe/meta.yaml (hunk 2 above)
Comment (mcg1969) [reply recipe/meta.yaml]: wow good catch! fixed

File: recipe/meta.yaml (hunk 3 above)
Comment (mcg1969) [reply recipe/meta.yaml:26]: thank you! I'll start doing this more deliberately now that I understand that.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**